### PR TITLE
 Added solutions from @mirpulatov, @Jlopezjlx, @Prrrince for calculator 

### PR DESCRIPTION
**Pull request by Jlopezjlx**
PR request, changes done

### CODE REVIEW

File: assignments/calculator/solutions/Jlopezjlx/calc/calc.py [hunk 1]
```diff
@@ -1,24 +1,40 @@
-print("CALCULADORA")
-print("OPCIONES: \n1-Suma\n2-Resta\n3-Multiplicar\n4-Dividir\n5-Restante")
+import maths
+loop = True
 
-opcion=int(input("Inserte opcion deseada: "))
-valor1 = int(input("Inserte primer valor: "))
-valor2 = int(input("Inserte segundo valor: "))
+while loop == True:
```
Comment (KenanBek) [inline assignments/calculator/solutions/Jlopezjlx/calc/calc.py:4]: Change this to:

```
while True:
```

and remove the `loop` variable. To stop the execution of the endless loop add one more operation (6-Exit) and when you get 6 as an operation stop the loop (example https://wiki.python.org/moin/WhileLoop).

File: assignments/calculator/solutions/Jlopezjlx/calc/calc.py [hunk 2]
```diff
@@ -1,24 +1,40 @@
-print("CALCULADORA")
-print("OPCIONES: \n1-Suma\n2-Resta\n3-Multiplicar\n4-Dividir\n5-Restante")
+import maths
+loop = True
```
Comment (KenanBek) [inline assignments/calculator/solutions/Jlopezjlx/calc/calc.py:2]: You do not need this, check comment below.

**Review by KenanBek — CHANGES_REQUESTED** [review]

File: assignments/calculator/solutions/Jlopezjlx/calc/calc.py (hunk 1 above)
Comment (Jlopezjlx) [reply assignments/calculator/solutions/Jlopezjlx/calc/calc.py:4]: Fixed

File: assignments/calculator/solutions/Jlopezjlx/calc/calc.py (hunk 2 above)
Comment (Jlopezjlx) [reply assignments/calculator/solutions/Jlopezjlx/calc/calc.py:2]: Fixed

**Comment by Jlopezjlx** [comment]
All is Fixed

**Comment by KenanBek** [comment]
@Jlopezjlx  did you update the PR?

**Pull request merged**